omnibus/files/private-chef-cookbooks/private-chef/recipes/opscode-erchef.rb

```diff
@@ -59,6 +59,12 @@
   notifies :restart, 'runit_service[opscode-erchef]' unless backend_secondary?
 end
 
+template "/opt/opscode/embedded/service/opscode-erchef/vm.args" do
```

Comment:
For sys.config we write the template into `/var/opt/opscode/opscode-erchef` and then symlink that into `/opt/opscode/embedded/service/opscode-erchef/`.  Perhaps we should do the same here for symmetry?